advancedstats.py

```diff
@@ -0,0 +1,187 @@
+import pandas as pd
+
+pd.set_option('display.max_rows', 500)
+pd.set_option('display.max_columns', 500)
+pd.set_option('display.width', 1000)
+
+def prepare_data(detailed_season_results_path):
+	'''
+	Returns dataframe that is suitable for performing group by.
+	It cleans any data to make it suitable for further processing
+	:param detailed_season_results_path:
+	:return:
+	'''
+	# Load the season results
+	seasons_results = pd.read_csv(detailed_season_results_path)
+
+	# Create a copy and rearrange column data
+	seasons_results_copy = seasons_results.copy()
+	seasons_results_copy[['WTeamID', 'WScore', 'WFGM', 'WFGA','WFGM3', 'WFGA3', 'WFTM', 'WFTA',	'WOR', 'WDR', 'WAst', 'WTO', 'WStl', 'WBlk', 'WPF']] = seasons_results[['LTeamID', 'LScore', 'LFGM', 'LFGA','LFGM3', 'LFGA3', 'LFTM', 'LFTA', 'LOR', 'LDR', 'LAst', 'LTO', 'LStl', 'LBlk', 'LPF']]
```

Comment:
its shorter if you make those column list into a variable and use it eg

```
win_stats_list = ['WTeamID', 'WScore' ...]
lose_stats_list = ['LTeamID', 'LScore' ... ]

seasons_results_copy[win_stats_list] = seasons_results[lose_stats_list]
```

i have something similar my pr